packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -457,15 +460,15 @@ export default class Lexis extends Module {
       }
 
       try {
-        homonym = await this.getWordQueryData({
+        wqData = await this.getWordQueryData({
           variables,
           source
         })
       } catch (err) {
         Logger.getInstance().error('Observable word query error', err)
       }
 
-      if (homonym) {
+      if (wqData && wqData.homonym) {
```

Comment:
If there is something returned in the `homonym` field, request succeeded fully or partially.